project_event/models/project_project.py

```diff
@@ -55,6 +55,13 @@ def _compute_event_log_count(self):
                 ('res_id', '=', rec.id)
             ])
 
+    @api.multi
+    def write(self, vals):
+        super(Project, self).write(vals)
+        for rec in self.task_ids:
+            rec.write({'responsible_id': str(self.responsible_id.id),
```

Comment:
Why str()?

Reply:
What if self.responsible_id (or parent_id) is empty?

Reply:
@rimbendhaou: Done